### PR TITLE
Optimize fps, grayscale, trim

### PR DESCRIPTION
**Pull request by Adib234**
## Summary
- [X] I have read CONTRIBUTING.md to understand how to contribute to this repository :)

<Please summarize what you are trying to achieve, what changes you made, and how they acheive the desired result.>
I'm trying to speed up the following video augmentations: fps, grayscale, and trim.

Here are the times which I measured with time for i in (seq 5); python -m unittest augly.tests.video_tests.transforms.ffmpeg_test.TransformsVideoUnitTest.test_{respective_augmentation};; end

Runtime with changes for FPS: 5.89s (test finished about 2s)
Runtime without changes for FPS: 11.592s (test finished about 7.5s)

Runtime with changes for Grayscale: 5.118s (test finished about 3s)
Runtime without changes for Grayscale: 14.132s (test finished about 10s)

Runtime with changes for Trim: 4.286s (test finished about 1.5s)
Runtime without changes for Trim: 10.072s (test finished about 5.5s)

## Unit Tests
If your changes touch the `audio` module, please run all of the `audio` tests and paste the output here. Likewise for `image`, `text`, & `video`. If your changes could affect behavior in multiple modules, please run the tests for all potentially affected modules. If you are unsure of which modules might be affected by your changes, please just run all the unit tests.

### Audio
```bash
python -m unittest discover -s augly/tests/audio_tests/ -p "*"
```

### Image
```bash
python -m unittest discover -s augly/tests/image_tests/ -p "*_test.py"
# Or `python -m unittest discover -s augly/tests/image_tests/ -p "*.py"` to run pytorch test too (must install `torchvision` to run)
```

### Text
```bash
python -m unittest discover -s augly/tests/text_tests/ -p "*"
```

### Video
```bash
python -m unittest discover -s augly/tests/video_tests/ -p "*"
```

### All
```bash
python -m unittest discover -s augly/tests/ -p "*"
```

## Other testing

If applicable, test your changes and paste the output here. For example, if your changes affect the requirements/installation, then test installing augly in a fresh conda env, then make sure you are able to import augly & run the unit test

### CODE REVIEW

File: augly/video/augmenters/ffmpeg/trim.py [hunk 1]
```diff
@@ -62,15 +60,17 @@ def add_augmenter(
...
-
-            return ffmpeg.concat(video, audio, v=1, a=1), {}
+        command = [
+            "-y",
+            "-i",
+            video_path,
+            "-vf",
+            f"trim={self.start}:{self.end}," + "setpts=PTS-STARTPTS",
+            "-af",
+            f"atrim={self.start}:{self.end}," + "asetpts=PTS-STARTPTS",
```
Comment (jbitton) [inline augly/video/augmenters/ffmpeg/trim.py:70]: have you ensured that this works for videos with no audio stream?

**Review by jbitton — COMMENTED** [review]
overall this lgtm, but have one comment:

File: augly/video/augmenters/ffmpeg/trim.py (hunk 1 above)
Comment (Adib234) [reply augly/video/augmenters/ffmpeg/trim.py:70]: This works https://github.com/facebookresearch/AugLy/pull/163#discussion_r766861727

**Review by jbitton — APPROVED** [review]
LGTM!

**Pull request closed without merging**